shifthelper/tools/whosonshift.py

```diff
@@ -27,10 +31,14 @@ def retrieve_shifters_from_calendar(
     calendar_entries["username"] = calendar_entries["u"]
 
     all_shifters = retrieve_valid_usernames_from_logbook()
-    tonights_shifters = pd.merge(all_shifters, calendar_entries, how='inner', on="username")
+    tonights_shifters = pd.merge(
+        all_shifters, calendar_entries,
+        how='inner', on="username"
```

Comment:
I personally like it better when one breaks a function call in to more lines, to explicitely state all args as kwargs and put each arg on one line.